### PR TITLE
fix infectee variables

### PR DESCRIPTION
**Pull request by pg2455**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #[ISSUE_NUMBER]

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [x] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: simulator.py [hunk 1]
```diff
@@ -669,20 +669,20 @@ def at(self, location, city, duration):
 
                 # cold_and_flu_transmission(self, h)
                 if self.cold_timestamp is not None or h.cold_timestamp is not None:
-                    infector, infectee = h, self
+                    cold_infector, cold_infectee = h, self
```
Comment (plstcharles) [inline simulator.py:672]: I'm not sure how these variables are reused anywhere in the function now; the change in variable name is not reflected in line 692 below

Comment (pg2455) [reply simulator.py:672]: infectee is being overwritten by these variables. `infectee` is being passed to the `logs`, so it is not logging the `covid infectee`

Comment (pg2455) [reply simulator.py:672]: Line #643 is where I had earlier defined `infectee` and passing it to the logs in line #692

**Review by plstcharles — APPROVED** [review]

**Comment by plstcharles** [comment]
@pg2455 you should force-merge this now, I'm pretty sure the tests will block on the silly 3.6 thing again

**Review by mweiss17 — APPROVED** [review]
ya I support force merging this

**Comment by satyaog** [comment]
3.6 test should be removed from the required tests:
settings/branches -> branch protection rules/master

**Comment by pg2455** [comment]
force merging!

**Pull request merged**